docs/consume-packages/Central-Package-Management.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Central Package Management
+description: Manage your dependencies in a central location and how you can get started with central package management.
+author: jondouglas
+ms.author: jodou
+ms.date: 2/25/2022
+ms.topic: conceptual
+---
+
+# Central Package Management
+
+Dependency management is a core feature of NuGet. Managing depdendencies for a single project can be easy. Managing dependencies for multi-project solutions can prove to be difficult as they start to scale in size and complexity. In situations where you manage common dependencies for many different projects, you can leverage NuGet's central package management features to do all of this from the ease of a single location.
+
+Historically, NuGet package dependencies have been managed in two main locations:
+
+- `packages.config` - An XML file used in older project types to maintain the list of packages referenced by the project.
+- `<PackageReference />` - An XML element used in new project types that manages NuGet dependencies directly within project files.
+
+Starting with [NuGet 7.0](..\release-notes\NuGet-7.0.md), you can centrally manage your dependencies in your projects with the addition of a `Directory.Packages.props` file.
+
+The feature is available across all NuGet integrated tooling.
+
+* [Visual Studio 2022 and later](https://visualstudio.microsoft.com/downloads/)
+* [.NET SDK 7.0.000 and later](https://dotnet.microsoft.com/download/dotnet/7.0)
+* [nuget.exe 7.0.0 and later](https://www.nuget.org/downloads)
+
+Older tooling will ignore Central Package Management configurations and features. To use this feature to the fullest extent, ensure all your build environments use the latest compatible tooling versions.
+
+Central Package Management will apply to all project types – including .NET Framework – as long as compatible tooling is used.
+
+## Enabling Central Package Management
+
+To get started with central package management, you can create a `Directory.Packages.props` file at the root of your solution.
+
+Inside, you can define each of the respective package versions required of your solution using `<PackageVersion />` elements that define the package ID and version.
+
+```xml
+<Project>
+  <ItemGroup>
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.1" />
+  </ItemGroup>
+</Project>
+```
+
+Within a project of the solution, you can then use the respective `<PackageReference />` syntax you know and love, but without a `Version` attribute to infer the centrally managed version instead.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" />
+  </ItemGroup>
+</Project>
+```
+
+Now you're using central package management and managing your versions in a central location!
```

Comment:
I think it'd be great if we can add a more detailed migration game, similar to what we did for source mapping. 

The rough steps would be: 

Run dotnet list packages or the equivalents. 
Start adding all top level dependencies in the Directory.Packages.props
Some guidance for resolving conflicts, including VersionOverride etc.
Remove *all* versions from PackageReference.